applications/working_with_helm_charts/configuring-custom-helm-chart-repositories.adoc

```diff
@@ -18,6 +18,14 @@ As a cluster administrator, you can add multiple cluster-scoped and namespace-sc
 
 As a regular user or project member with the appropriate role-based access control (RBAC) permissions, you can add multiple namespace-scoped Helm chart repositories, apart from the default cluster-scoped Helm repository, and display the Helm charts from these repositories in the *Developer Catalog*.
 
+In the *Developer* perspective of the web console, you can use the *Helm* page to: 
+
+* Create Helm Releases and Repositories using the *Create* button.
+
+* Create, update, or delete a Cluster Scoped or Namespace Scoped Helm chart repository. 
+
+* View the list of the existing Helm chart repositories in the Repositories tab, which can also be easily distinguished as either Cluster Scoped or Namespace Scoped.
```

Comment:
```suggestion
* View the list of the existing Helm chart repositories in the Repositories tab, which can also be easily distinguished as either cluster scoped or namespace scoped.
```

Reply:
But not hyphenated here because they don't precede a noun.

Reply:
Fixed.